src/main/java/game/GameEntity.java

```diff
@@ -0,0 +1,23 @@
+package game;
+
+public class GameEntity {
+    private int guessCount;
+    private int generatedNumber;
+
+    public GameEntity(){
+        guessCount = 0;
+        generatedNumber = 0;
+    }
+
+    public void setGuessCount(int guessCount){ this.guessCount = guessCount; }
```

Comment:
S**O**LID
Open for extension, closed for modification - jokių setterių!

Reply:
man rodos open for extension, closed for modification nereiškia kad negalima keist objekto būsenos.
"The general idea of this principle is great. It tells you to write your code so that you will be able to add new functionality without changing the existing code. That prevents situations in which a change to one of your classes also requires you to adapt all depending classes. "
https://stackify.com/solid-design-open-closed-principle/